app/styles/ui/rebase-status.scss

```diff
@@ -0,0 +1,68 @@
+.rebase-status-component {
+  .rebase-status-icon-container {
+    height: 20px;
+    position: relative;
+    text-align: center;
+
+    .rebase-status {
+      background: var(--background-color);
+      border-width: 0 var(--spacing-half);
+      border: solid transparent;
+      box-sizing: content-box;
+      position: relative;
+      z-index: 1;
```

Comment:
I'm really not a big fan of us using `z-index` unless it's absolutely necessary.  I'm pretty sure that if we removed the relative positioning and use a Flexbox this would not be necessary

Reply:
As above, we have the same problem in `app/styles/ui/_merge-status.scss` - I'm happy to leave this to someone with some bandwidth and CSS skills to figure out if it doesn't need to hold up this PR.

Reply:
Same thing as the previous CSS question - @donokuda can you recall why we settled on this?

Reply:
> I'm happy to leave this to someone with some bandwidth and CSS skills to figure out if it doesn't need to hold up this PR.

:+1: Sounds good